### PR TITLE
Feature/cursor effect

### PR DESCRIPTION
**Pull request by jjy331**
Portfolio mouse cursor style and animation

### CODE REVIEW

File: pfjjy/styles/globals.css [hunk 1]
```diff
@@ -21,16 +21,21 @@ body {
 .cursor {
   width: 0.5rem;
   height: 0.5rem;
-  backdrop-filter: invert(1);
```
Comment (jjy331) [inline pfjjy/styles/globals.css]: error_log

1. safari_cross_browsing bug 
> backdrop-filter: invert(1); not working 
> solution: filter: invert(1);

File: pfjjy/pages/cursor.js [hunk 2]
```diff
@@ -73,14 +73,19 @@ export default function CustomCursor() {
...
-        <div
-            className={cursorClasses}
-            style={{ left: `${position.x}px`, top: `${position.y}px` }}
-        />
+        // <div
+        //     className={cursorClasses}
+        //     style={{ left: `${position.x}px`, top: `${position.y}px` }}
+        // />
+        <div>
+            <svg className={cursorClasses} style={{ left: `${position.x}px`,top: `${position.y}px` }}>
```
Comment (jjy331) [inline pfjjy/pages/cursor.js:86]: error_log

It was created to solve Safari's transform:scale issue.
But it does not work.
it's still blurry

ref
https://discourse.webflow.com/t/problem-with-scaling-in-safari/87535﻿

File: pfjjy/styles/globals.css [hunk 3]
```diff
@@ -41,13 +46,12 @@ body {
   transform: translate(-50%, -50%) scale(3.8);
```
Comment (jjy331) [inline pfjjy/styles/globals.css:46]: On a circle, if the transform:scale is bigger than before, the border will be blurry.
> solution: box-shadow: 0px 0px 0px 100px black 

Rectangles have less blur.

**Review by jjy331 — COMMENTED** [review]
Issue 

1. safari cross browsing
> darkdrop-filter 
> transform: scale animation blur 

 Solutions

1. Find a solution to the bug.
2. Bypass the bug and change the design to proceed with the work.

**Review by jjy331 — COMMENTED** [review]
Issue 

1. safari cross browsing
> darkdrop-filter 
> transform: scale animation blur 

 Solutions

1. Find a solution to the bug.
2. Bypass the bug and change the design to proceed with the work.

**Review by jjy331 — COMMENTED** [review]
Issue 

1. safari cross browsing
> darkdrop-filter 
> transform: scale animation blur 

 Solutions

1. Find a solution to the bug.
2. Bypass the bug and change the design to proceed with the work.

**Review by jjy331 — COMMENTED** [review]
Issue 

1. safari cross browsing
> darkdrop-filter 
> transform: scale animation blur 

 Solutions

1. Find a solution to the bug.
2. Bypass the bug and change the design to proceed with the work.

File: pfjjy/pages/cursor.js (hunk 2 above)
Comment (jjy9331) [reply pfjjy/pages/cursor.js:86]: ref 

https://greensock.com/forums/topic/11692-transform-scaled-svg-causes-blur/

**Pull request merged**